frontend/src/App.js

```diff
@@ -1,42 +1,30 @@
 import React, { Component } from 'react';
 import logo from './logo.png';
 import './App.css';
-
-const headers = new Headers({
-  Accept: 'application/json',
-  'Content-Type': 'application/json',
-});
+import Home from './Home.jsx';
+import Experiment from './Experiment.jsx';
+import Status from './Status.jsx';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 
 class App extends Component {
-  componentWillMount() {
-    this.testBackend();
-  }
-
-  state = {
-    test: 'nothing.',
-  }
-
-  testBackend = async () => {
-    const test = await fetch('http://localhost:80/api/', {
-      method: 'GET',
-      headers,
-      cache: 'default',
-    });
-    const testJson = await test.json();
-    if (testJson && testJson.status) {
-      this.setState({ test: testJson.status });
-    }
-  }
-
   render() {
```

Comment:
Here you have an *eslint* error:
![image](https://user-images.githubusercontent.com/23021242/51538240-ca190400-1e1e-11e9-9422-50695f7758e5.png)

Can we also fix that one?